src/sass/_patterns-global-nav.scss

```diff
@@ -1,74 +1,155 @@
+$global-asset-path: '//assets.ubuntu.com/v1/';
+
 @mixin global-p-nav {
   body {
-    padding-top: 30px;
+    padding-top: 1.875rem;
+
+    @media only screen and (max-width: 768px) {
+      padding-bottom: 1.875rem;
+      padding-top: 0;
+    }
   }
 
   .global-nav {
     background: $global-background;
-    width: 100%;
-    height: 30px;
+    box-shadow: 0 2px 2px transparentize($global-black, .8);
     display: block;
-    z-index: 101;
-    position: absolute;
-    top: 0;
+    height: 1.875rem;
     left: 0;
-    box-shadow: 0 2px 2px rgba(0, 0, 0, .2);
     margin: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: 101;
+
+    @media only screen and (max-width: 768px) {
+      background: $global-transparent;
+      bottom: 0;
+      box-shadow: none;
+      top: auto;
+    }
```

Comment:
I think the overall positioning needs a rethink, when adding a lot of content it'll sit in the middle of any content there. I feel we should be using a different approach here. 

Here is a screenshot showing the position when there are 20 paragraph elements on the page.
<img width="2032" alt="screen shot 2017-05-02 at 16 26 50" src="https://cloud.githubusercontent.com/assets/1201618/25625575/c98cf434-2f54-11e7-8cc9-816232572e01.png">